src/components/layers-panel/layer-manager/LayerManager.ts

```diff
@@ -39,7 +39,7 @@ export class LayerManager extends i18nMixin(LitElement) {
 
   sortMethod(event: SortableEvent) {
     const items = event.to.children
-    mapState.reorderLayers([...items].map(val => Number(val.id)))
+    mapState.reorderLayers(Array.from(items).map(val => Number(val.id)))
```

Comment:
I had a TS error : `Type 'HTMLCollection' must have a '[Symbol.iterator]()' method that returns an iterator.` thus the `Array.from()`.
But I have canceled my change and added `"dom.iterable"` in tsconfig file to use spread with HTMLCollection instead of using Array.from().

Reply:
oh ok ! :)
yes `HTMLCollection` is not an Array sorry